src/exchange/ExchangeScreenHeader.tsx

```diff
@@ -76,7 +77,15 @@ function ExchangeTradeScreenHeader({ currency, isCeloPurchase, onChangeCurrency
 
   return (
     <>
-      <CustomHeader left={<BackButton eventName={cancelEventName} />} title={title} />
+      <CustomHeader
+        left={
+          <BackButton
+            style={{ paddingLeft: variables.contentPadding }}
```

Comment:
Looks like this is added in multiple places, is there a way to hide this detail? so we don't have to modify multiple place.

Reply:
Headers would make a good follow up PR.

Reply:
can't we just add the `paddingLeft: variables.contentPadding` style to the `BackButton` directly, and allow the CustomHeader to overwrite this style? then we don't need to include this style as inline everywhere

Reply:
I tried adding it directly to the back button and it makes it so there are additional changes in other screens already using the BackButton component.

Reply:
Adjusted the header changes in c4fd878f8760e1507784e5888c8e8086bd4eba2b. There's a few specific changes to headers in this PR but they should be easier to follow now.